File: src/app/entity/Address.ts
export class Address {

    private id : number;

    private streetName : string;

    private number : string;

    private complement : string;

    private zipcode : string;

    private city : string;

    private state : State;

    private geolocationstate : Geolocation;

    constructor() { }

    getId() : number {
        return this.id;
    }

    getStreetName() : string {
        return this.streetName;
    }

    getNumber() : string {
        return this.number;
    }

    getComplement() : string {
        return this.complement;
    }

    getZipcode() : string {
        return this.zipcode;
    }

    getCity() : string {
        return this.city;
    }

    getState() : State {
        return this.state;
    }

    getGeolocationstate() : Geolocation {
        return this.geolocationstate;
    }

  
}

File: petlife-services/petlife-services-boot/src/main/java/br/com/duti/petlife/models/Address.java
package br.com.duti.petlife.models;

import java.io.Serializable;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.EnumType;
import javax.persistence.Enumerated;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

//@Table(name="address")
//@Entity
public class Address implements Serializable {
	
	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;
	
	//@Id
	//@GeneratedValue(strategy = GenerationType.IDENTITY)
	private int id;
	
	//@Column(name="street_name", nullable=false)
	private String streetName;
	
	//@Column(name="number", nullable=false)
	private String number;
	
	//@Column(name="complement", nullable=true)
	private String complement;
	
	//@Column(name="zipcode", nullable=false)
	private String zipcode;
	
	//@Column(name="city", nullable=false)
	private String city;
	
	//@Enumerated(EnumType.STRING)
	//@Column(name="state", nullable=false, length=2)
	private State state;

	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}

	public String getStreetName() {
		return streetName;
	}

	public void setStreetName(String streetName) {
		this.streetName = streetName;
	}

	public String getNumber() {
		return number;
	}

	public void setNumber(String number) {
		this.number = number;
	}

	public String getComplement() {
		return complement;
	}

	public void setComplement(String complement) {
		this.complement = complement;
	}

	public String getZipcode() {
		return zipcode;
	}

	public void setZipcode(String zipcode) {
		this.zipcode = zipcode;
	}

	public String getCity() {
		return city;
	}

	public void setCity(String city) {
		this.city = city;
	}

	public State getState() {
		return state;
	}

	public void setState(State state) {
		this.state = state;
	}
	
	//private geolocationstate : Geolocation;
}
